frontend/locales/en/strings.json

```diff
@@ -13,7 +13,20 @@
       "title": "$t(global.projectName) documentation",
       "link": "https://pcluster.cloud/"
     },
-    "openNewTab": "Open in a new tab"
+    "openNewTab": "Open in a new tab",
```

Comment:
todo: use `openNewTab` from global section

Reply:
Great catch, thank you!